<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="InGame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>In Game</string>
   </property>
   <widget class="QSpinBox" name="Difficulty">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Difficulty_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FOV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FOV">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Display FPS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DisplayFPS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>General Settings</string>
   </property>
   <widget class="QCheckBox" name="Windowed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Windowed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FullScreen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Screen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Resolution">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Rate Cap</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FramRateCap">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="Cap">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>421</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Key Binds</string>
   </property>
   <widget class="QLabel" name="Up_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="Left_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="Jump_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump</string>
    </property>
   </widget>
   <widget class="QLabel" name="Down_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="Right_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Up">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Down">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Left">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Right">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Jump">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
